client/signup/site-mockup/index.jsx

```diff
@@ -70,15 +70,7 @@ class SiteMockups extends Component {
 		return parts.slice( 0, 2 ).join( ', ' );
 	}
 
-	shouldRender() {
```

Comment:
We only want to display the site preview for business sites. This could be a separate PR though to make the reviews smoother since we're only updating the `onboarding-dev` flow.

Reply:
> We only want to display the site preview for business sites. 

Thanks for the review! This PR enables such logic control but at a higher level in `flows-pure`. 

It does this by giving us the power to define flows _specifically_ for business. 

For example, when a user selects **Business** at the site style step, we can redirect to a flow that contains both the preview and the site style step.

It probably wouldn't hurt to be more defensive however so that, should a user land on this flow somehow with an undefined or non-Business site type, we shunt them back to the site type step. I'll explore in another PR, as you mentioned. 👍